maxim/logger/components/utils.py

```
@@ -0,0 +1,81 @@
+    Attachment,
+    FileDataAttachment,
+    UrlAttachment,
+)
+
+if TYPE_CHECKING:
+    from .generation import (
+        GenerationRequestMessage,
+    )
+
+
+def parse_attachments_from_messages(
```

Ruff (0.11.9)

`parse_attachments_from_messages` is too complex (12 > 10)

(C901)